### ISSUE TITLE
[ux] Clickable elements in notification widget should be <a> elements

### ISSUE THREAD

**Issue by nemesisdesign** [openwisp/openwisp-notifications#122]
Clickable elements in notification widget should be `<a>` elements for reasons of accessibility.

Each notification is a `<p>` which contains an `<a>`, the `<p>` could become `<a>` and the `<a>` could become something neutral like <span>.

**Comment by pandafy** [comment]
The notification message HTML is rendered from markdown. Maybe use `aria` attributes?

**Comment by nemesisdesign** [comment]
> The notification message HTML is rendered from markdown. Maybe use `aria` attributes?

Yes, that could work!
